### PR TITLE
storagenode/consoledb: fix daily bandwidth query

### PR DESCRIPTION
**Pull request by rikysya**
What:  storagenode/consoledb fix daily bandwidth query

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: storagenode/storagenodedb/consoledb.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func (db *consoledb) getDailyBandwidthUsed(ctx context.Context, cond string, arg
 	qb.WriteString("GROUP BY DATE(created_at), action ")
 	qb.WriteString("ORDER BY created_at ASC")
 
-	rows, err := db.db.QueryContext(ctx, db.Rebind(qb.String()), args)
+	rows, err := db.db.QueryContext(ctx, db.Rebind(qb.String()), args...)
 	if err != nil {
```
Comment (egonelbre) [inline storagenode/storagenodedb/consoledb.go]: This seems to be missing tests.

File: storagenode/storagenodedb/consoledb.go [hunk 2]
```diff
@@ -98,7 +98,7 @@ func (db *consoledb) getDailyBandwidthUsed(ctx context.Context, cond string, arg
 	qb.WriteString("GROUP BY DATE(created_at), action ")
 	qb.WriteString("ORDER BY created_at ASC")
```
Comment (egonelbre) [inline storagenode/storagenodedb/consoledb.go]: Using a string addition and multiline string would be completely fine here it doesn't make a big difference.

Comment (rikysya) [reply storagenode/storagenodedb/consoledb.go]: It looks more clear this way and easier to make changes imo

Comment (egonelbre) [reply storagenode/storagenodedb/consoledb.go]: Not just a nit, nothing needs to change. The difference I see:

```
qb := strings.Builder{}
qb.WriteString("SELECT action, SUM(amount), created_at ")
qb.WriteString("FROM bandwidth_usage ")
if cond != "" {
    qb.WriteString(cond + " ")
}
qb.WriteString("GROUP BY DATE(created_at), action ")
qb.WriteString("ORDER BY created_at ASC")

rows, err := db.db.QueryContext(ctx, db.Rebind(qb.String()), args...)
if err != nil {
    return nil, err
}
```

compared to:

```
    query := db.Rebind(`
        SELECT action, SUM(amount), created_at
        FROM bandwidth_usage
        ` + cond + `
        GROUP BY DATE(created_at), action 
        ORDER BY created_at ASC
    `)

    rows, err := db.db.QueryContext(ctx, query, args...)
	if err != nil {
		return nil, err
	}
```

File: storagenode/storagenodedb/consoledb.go (hunk 1 above)
Comment (rikysya) [reply storagenode/storagenodedb/consoledb.go]: let's make it a separate pull request, so the other part of the team can use the query meanwhile

**Pull request merged**